### PR TITLE
Add Badge component

### PR DESCRIPTION
**Pull request by sondrealf**
Tror styling component er ferdig. Men klarer ikke sjekke ut stylingen i storyboard.
Dark mode til storyboard er også fjernet.

### CODE REVIEW

File: src/components/atoms/badge/badge.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { Badge, BadgeProps } from "theme-ui";
+
+export default {
+    title: "atoms/Badge",
+    component: Badge,
+};
+
+export const subtleBadge = (props: BadgeProps) => <Badge variant="test">TEST</Badge>
```
Comment (supergrecko) [inline src/components/atoms/badge/badge.stories.tsx:8]: Is this supposed to have the `test` variant?

File: src/components/atoms/badge/badge.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import { Badge as TBadge } from "theme-ui";
+
+interface BadgeProps extends ComponentProps<typeof TBadge> {
+  color: string;
+}
+
+const Badge: FC<BadgeProps> = ({ children, color, ...props }) => <TBadge {...props}> {children} </TBadge>;
+
+export default Badge;
+*/
```
Comment (supergrecko) [inline src/components/atoms/badge/badge.tsx:12]: Remove these lines

File: src/styles/components/badge.ts [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+      bg: darken(i, 0.02),
+    },
+    outline: {
+      variant: "badges.base",
+      color: darken(i, 0.1),
+      bg: lighten(i, 0.6),
+    },
+  };
+}
+*/
```
Comment (supergrecko) [inline src/styles/components/badge.ts:68]: Remove these lines

**Review by supergrecko — CHANGES_REQUESTED** [review]

File: src/components/atoms/badge/badge.stories.tsx (hunk 1 above)
Comment (sondrealf) [reply src/components/atoms/badge/badge.stories.tsx]: Not really, used test variant to check if I could see the styling component in storyboard, but i cant and i dont know why.

**Review by kharann — CHANGES_REQUESTED** [review]
Delete package-lock :^)

**Comment by kharann** [comment]
There also seems to be a bug because on the [storybook](https://6187e4eb0c4f95004a20f76d-rnqnmvjqci.chromatic.com/) all the badges look the same.

**Review by supergrecko — CHANGES_REQUESTED** [review]
Did you mean to delete the yarn.lock? The package-lock.json should be removed.

**Comment by supergrecko** [comment]
It looks like your patch bumped every single dependency up to their latest semver patch version, doesn't seem like it was intended?

**Comment by supergrecko** [comment]
Closed in favor of #49

**Pull request closed without merging**